AllReadyApp/Web-App/AllReady.UnitTest/Controllers/TaskApiControllerTests.cs

```diff
@@ -77,7 +77,7 @@ public class TaskApiControllerTests
             var mediator = new Mock<IMediator>();
             mediator.Setup(x => x.SendAsync(It.IsAny<EventByEventIdQuery>())).ReturnsAsync(new Event());
             mediator.SetupSequence(x => x.SendAsync(It.IsAny<TaskByTaskIdQuery>()))
-                .ReturnsAsync(new AllReadyTask()).ReturnsAsync(null);
+                .ReturnsAsync(allReadyTask).ReturnsAsync(null);
```

Comment:
@stevejgordon Does this compile for you? I am getting this error at this line. Not sure whether I am missing something
    `Error	CS1061	'ISetupSequentialResult<Task<AllReadyTask>>' does not contain a definition for 'ReturnsAsync' and no extension method 'ReturnsAsync' accepting a first argument of type `